CMakeLists.txt

```diff
@@ -299,13 +302,15 @@ else()
     find_program(EXTRACTKERNEL_BIN extractkernel
         PATH_SUFFIXES bin
         PATHS
+            ${ROCM_ROOT}/hip
+            ${ROCM_ROOT}/hcc
+            ${ROCM_ROOT}
```

Comment:
Remove ROCM_ROOT variable, the CMAKE_PREFIX_PATH(or CMAKE_INSTALL_PREFIX) should be used when rocm is not installed into /opt/rocm or standard directories(ie /usr or /usr/local).

Reply:
Done.